corehq/apps/userreports/templates/userreports/reportbuilder/configure_report.html

```diff
@@ -92,7 +92,7 @@ <h3 data-bind="text: reportType() === 'agg' ?  '{% trans 'Indicators' %}' : '{%
               <td>
                 <select data-bind="fadeVisible: $parent.isAggregationEnabled,
                                    value: groupByOrAggregation">
-                  <option value="groupBy">{% trans 'Display Each' %}</option>
+                  <option value="groupBy">{% trans 'Group By' %}</option>
```

Comment:
I wasn't sure at first but I think we settled on "Group By". See here: https://docs.google.com/document/d/133FssY68L2tS9pu6WbWEzS0NlE67yuWfT3-daEbvKoc/edit#heading=h.fu2426qi6ssd